### PR TITLE
Home

### PR DESCRIPTION
**Pull request by saketkum11**
- Html basics Home page 
- Routes
- Header

### CODE REVIEW

File: src/Pages/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      <main className="wt-100 m-auto ">
+        <section className=" bg-black-8  wt-100   flex ">
+          <div className="  text-color-0 flex wt-80 m-auto justify-center pd-11">
+            <div className="position-rel">
+              <img
+                src="./assets/farming.jpg"
+                className="object-content wt-80 rounded-m"
+                alt=""
+              />
+              <FaPlayCircle className="fa-3x  fa-5x position-ab text-color-5 icon-position"></FaPlayCircle>
```
Comment (MiheerTamkhane) [inline src/Pages/Home/Home.jsx:18]: Use self closing tag here : <FaPlayCircle />

**Review by narharikale — APPROVED** [review]
good  to go ✔

**Review by Ash-D23 — APPROVED** [review]
Good Folder structure, looks clean

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                  alt=""
+                />
+              </div>
+
+              <div className="pd-x-4 flex  flex-column">
+                <div className=" flex ">
+                  <span className="text-s flex items-start">
+                    Palm Oil Farming | सरकार उठाएगी खर्च | पाम तेल की खेती |
+                    Palm oil tree | Palm farming subsidy
+                  </span>
```
Comment (MiheerTamkhane) [inline src/Pages/Home/Home.jsx:93]: This info you should get from database. If you are going to do that next. I'll suggest do it first then make all things.

File: src/Pages/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* My Page and component*/
+export { Home } from "../Pages/Home/Home";
+export { Header } from "../Component/Header/Header";
+export { VideoList } from "./VideoPlayList/VideoList";
+export { History } from "../Pages/History/History";
```
Comment (MiheerTamkhane) [inline src/Pages/index.js:5]: Good work here

**Review by MiheerTamkhane — APPROVED** [review]
All things are looking good to me. Also code is readable and well structured

File: src/Pages/Home/Home.jsx (hunk 2 above)
Comment (saketkum11) [reply src/Pages/Home/Home.jsx:93]: yes after thus  will give to data base.

File: src/Pages/index.js (hunk 3 above)
Comment (saketkum11) [reply src/Pages/index.js:5]: Your last time suggestion I appiled here

File: src/Pages/Home/Home.jsx (hunk 1 above)
Comment (saketkum11) [reply src/Pages/Home/Home.jsx:18]: sure

**Review by shubhamkumar648 — COMMENTED** [review]
Good folder structure and break component , good to merge

**Pull request merged**